src/kirby/components/avatar/avatar.component.tns.ts

```diff
@@ -1,10 +1,8 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, AfterViewInit, ContentChild, AfterContentInit } from '@angular/core';
```

Comment:
Looks like unused imports.

Reply:
Could we do automatic tslint --fix as a pre-commit hook? :)

Reply:
Removed now :)